src/util.js

```diff
@@ -136,11 +138,16 @@ module.exports.cleanup = function cleanup(config) {
       const json = {
         name: createTestPlan ? resultSet[0].browserName : options.title,
         suite_id: options.suiteId,
-        description,
+        description
       };
       if (options.includeAll === false) {
         json.include_all = false;
-        json.case_ids = results.map((currentResult) => currentResult.case_id);
+        if (options.casesFieldFilter) {
```

Comment:
Does it work without this if statement?  It looks like results is already filtering for actualCaseIds [on line 78](https://github.com/billylam/wdio-wdiov5testrail-reporter/blob/f2883d7148f95150461f1152cdf85d55e7e9c08f/src/util.js#L78)

Reply:
Oh ok, thanks for the explanation.  I’ll merge and deploy later today.